bin/build_aggregate.sh

```diff
@@ -1,48 +0,0 @@
-######################
```

Comment:
👎 

This build script still has some use. I maybe want to somehow modify it in light of the SPM introduction, but I still want to keep it present. This script allows you to build a library that will work in the Simulator program regardless of the target platform you select. As such, it's very useful during development and platform testing.